src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -198,6 +198,7 @@ int sd_dhcp_server_new(sd_dhcp_server **ret, int ifindex) {
         server->address = htobe32(INADDR_ANY);
         server->netmask = htobe32(INADDR_ANY);
         server->ifindex = ifindex;
+        server->emit_router = false;
```

Comment:
Once again, `new0` initializes the structure to all zeros, so this is redundant. You'll notice that only things which need to be non-zero are initialized here.